golem.yaml

```
@@ -56,6 +56,8 @@ subpackages:
           mkdir -p ${{targets.contextdir}}/
           mkdir -p ${{targets.contextdir}}/config
           mkdir -p ${{targets.contextdir}}/db
+
+          mv ./target/release/golem-component-service ${{targets.contextdir}}/usr/bin/golem-component-service
```

```suggestion
          install -Dm0755 ./target/release/golem-component-service ${{targets.contextdir}}/usr/bin/golem-component-service
```
You can use "install" to get this binary in the right place, it will be a copy + changes the file attributes to whatever you specify on that `-m` flag, in this case we can install + make the binary executable with `-Dm0755` -> creates all the dirs, then makes the file executable